assets/_scss/elements/_inputs.scss

```diff
@@ -138,3 +138,98 @@ input[type="radio"]:disabled + label::before {
   color: $color-grey-600;
   cursor: not-allowed;
 }
+
+// Range inputs
+
+input[type=range] {
+  -webkit-appearance: none;
+  border: none;
+  margin: 5.5px 0;
+  padding-left: 0;
+  width: 100%;
+}
+
+input[type=range]:focus {
+  box-shadow: none;
+  outline: none;
+}
+
+input[type=range]::-webkit-slider-runnable-track {
+  background: $color-primary;
+  border: 1px solid $color-grey-600;
+  border-radius: 10px;
+  cursor: pointer;
+  height: 9px;
+  width: 100%;
+}
+
+input[type=range]::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  border: 1.8px solid $color-grey-600;
+  height: 20px;
+  border-radius: 15px;
+  background: #eeeeee;
+  cursor: pointer;
+  margin-top: -6.5px;  
+  width: 20px;
+}
+
+input[type=range]:focus::-webkit-slider-runnable-track {
```

Comment:
Are we going to be able to replace these with generics when we add auto-prefixer? Or are these stuck as browser prefixes?